lib/perl/Genome/FeatureList.pm

```diff
@@ -421,11 +421,17 @@ sub generate_converted_bed_file {
     }
     my $original_md5 = Genome::Sys->md5sum($original_file_path);
 
+    my $result_users = delete($args{result_users});
+    $result_users->{'feature-list'} = $self;
+    $result_users->{requestor}    ||= $reference;
+    $result_users->{sponsor}      ||= Genome::Sys->current_user;
```

Comment:
These defaults are present so that the `genome feature-list dump-merged-list` command can still function without requiring the end user to say they're making a BED file for a particular build and AnP.

I don't think it's worse than setting the `requestor` to the reference sequence build, but I agree that both are not great.  Maybe we do force users to provide this info?